macros/sql/unpivot.sql

```diff
@@ -23,22 +27,26 @@ Arguments:
   {%- set cols = adapter.get_columns_in_relation(table) %}
 
   {%- for col in cols -%}
-    {%- if col.column.lower() not in exclude|map('lower') -%}
+    {%- if col.column.lower() not in remove|map('lower') and col.column.lower() not in exclude|map('lower') -%}
       {% set _ = include_cols.append(col) %}
     {%- endif %}
   {%- endfor %}
 
-  {%- for col in include_cols -%}
 
+  {%- for col in include_cols -%}
     select
       {%- for exclude_col in exclude %}
         {{ exclude_col }},
       {%- endfor %}
-      cast('{{ col.column }}' as {{ dbt_utils.type_string() }}) as field_name,
-      cast({{ col.column }} as {{ cast_to }}) as value
+
+      cast('{{ col.column }}' as {{ dbt_utils.type_string() }}) as "{{ field_name }}",
+      cast({{ col.column }} as {{ cast_to }}) as "{{ value_name }}"
```

Comment:
```suggestion
      cast({{ col.column }} as {{ cast_to }}) as {{ value_name }}
```
Same as above - let's skip the quotes and defer the quoting responsibilities to the caller. You can always do:
```
{{ unpivot(table=ref('users'), field_name='"my=quoted-value"') }}
```
if quotes are necessary in the resulting column names

Reply:
got it @drewbanin